ouroboros/__builtins__.py

```diff
@@ -135,6 +135,30 @@ def __exit__(self, type, value, tb):
 
 sentinel = object()
 
+#min
+def min(object):
+	"""
+		This method will take an iterator and return the minimum value from there.
+	"""
+	if type(object) is int:
+		raise TypeError("'int' object is not iterable")
+	elif type(object) is float:
+		raise TypeError("'float' object is not iterable")
```

Comment:
This hardcodes 2 specific types, rather than checking for iterability. What happens if I pass in an object that *does* implement `__len__`, but *doesn't* implement `__iter__`?

Reply:
Added check for __iter__ then hard-coding objects.